plugins/hu.bme.mit.gamma.yakindu.genmodel.language/src/hu/bme/mit/gamma/yakindu/genmodel/language/validation/GenModelValidator.xtend

```diff
@@ -420,11 +420,11 @@ class GenModelValidator extends AbstractGenModelValidator {
 					}
 					return yEvent.type.name.equals("boolean")					
 				}
-				RealTypeDefinition: {
+				DecimalTypeDefinition: {
 					if (yEvent.type === null) {
 						return false
 					}
-					return yEvent.type.name.equals("real")					
```

Comment:
I think this line should remain the original, as Yakindu refers to the type as "real" and not "decimal". It would be great if you could check it by declaring a real variable in one of the transformable Yakindu models and see whether you get an error marker or not.